components/style/properties/gecko.mako.rs

```diff
@@ -2378,6 +2378,11 @@ fn static_assert() {
     }
 
     #[allow(non_snake_case)]
+    pub fn reset__x_lang(&mut self, other: &Self) {
```

Comment:
nit: We don't change anything here and the new methods will be removed after my comment below. So we can put that back to its previous place.